public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta name="theme-color" content="#000000" />
+    <meta
+      name="description"
+      content="Web site created using create-react-app"
```

Comment:
Is this entire pull request the raw, unedited result of running a setup command? If so that should be stated somewhere. 

Still, seems like basic strings like this should be set (or make it clear that you'll be setting them in another pull request)

Reply:
Yes, this whole pull request is purely from the setup command. I was unsure how to word it properly, but this is essentially the foundations that we all will be working from